tools/test-framework/src/chain/cli/transfer.rs

```diff
@@ -38,3 +38,48 @@ pub fn local_transfer_token(
 
     Ok(())
 }
+
+pub fn transfer_with_memo(
+    chain_id: &str,
+    command_path: &str,
+    home_path: &str,
+    rpc_listen_address: &str,
+    sender: &str,
+    recipient: &str,
+    token: &str,
+    channel: &str,
+    port: &str,
+    memo: &str,
+) -> Result<(), Error> {
+    simple_exec(
```

Comment:
Once this is working, we should update `ibc-proto-rs` to include the memo field, and have another PR to update this method to submit the transaction within Rust.